Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -284,3 +284,11 @@
   category: Misc
   itemId: ClothingBackpackDuffelSyndicatePyjamaBundle
   price: 4
+
+- type: uplinkListing
+  id: UplinkGigacancerScanner
+  category: Misc
+  itemId: HandheldHealthAnalyzerGigacancer
+  listingName: Gigacancer Health Analyzer
+  description: Works like a normal health analyzer other than giving everyone it scans gigacancer.
```

```suggestion
  description: Works like a normal health analyzer, other than giving everyone it scans gigacancer.
```

maybe get some input on something better than 'gigacancer'